common/containers/Tabs/Swap/index.js

```diff
@@ -0,0 +1,120 @@
+import React, {Component} from 'react';
+import WantToSwapMy from './components/wantToSwapMy';
+import CurrentRates from './components/currentRates';
+import {connect} from 'react-redux';
+import {
+    SWAP_DESTINATION_AMOUNT_TO,
+    SWAP_DESTINATION_KIND_TO,
+    SWAP_ORIGIN_AMOUNT_TO,
+    SWAP_ORIGIN_KIND_TO,
+    SWAP_UPDATE_BITY_RATES_TO
+} from 'actions/swap';
+
+import PropTypes from 'prop-types';
+import Bity from 'api/bity';
+
+class Swap extends Component {
+    constructor(props) {
+        super(props);
+        this.bity = new Bity();
+    }
+
+    static propTypes = {
+        bityRates: PropTypes.any,
+        originAmount: PropTypes.any,
+        destinationAmount: PropTypes.any,
```

Comment:
Sorta. `originAmount` and `destinationAmount` can be either strings or ints, so I said any. bityRates is an object, I can fix that.